### PR TITLE
Remove unnecessary check in add_agent_pos!

### PR DESCRIPTION
**Pull request by Tortar**
This will fail with an easy to bisect error anyway since adding to the model will fail since the agent type will not be accepted. Forgot to do it when merging #900

### CODE REVIEW

**Comment by Tortar** [comment]
wops! sorry renamed one of the files before merging without changing the including statement!!

**Comment by Tortar** [comment]
For some reason, I can't anymore bypass the protection rule otherwise I would have merged this one

**Comment by Datseris** [comment]
@Tortar I've updated the rules of Agents.jl to ensure high quality of contributions.

1. Can't push directly to master, must do PR.
2. PRs must have one accepting review from someone other than the creator.

Things might get slower, but they are safer this way.

**Review by Datseris — APPROVED** [review]

**Pull request merged**
